### PR TITLE
MBM-419 Item pages

### PR DESCRIPTION
**Pull request by insertStringHere**
This reverts commit a41d1b426a161ca998996b7b6700598c92c7c865.

### CODE REVIEW

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/myrial_diamond_ingot.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Myrial Diamond Ingot",
+    "icon": "minebound:myrial_diamond_ingot",
+    "category": "minebound:alloys",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "$(o)Fusing diamond with the $(l:minebound:myrial_steel_ingot)myrial miracle metal$()$(o) results in an alloy on the same level as netherite...who would have guessed? That said, thanks to the presence of myrial steel, these bars are far more suited for conducting mana than netherite, making them the perfect final step for your $(l:minebound:magic/myrial_armors)quest$()$(o) to make yourself as powerful as a myri."
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/myrial_diamond_ingot.json:8]: Both the links in this text are bad.

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/illuminant_steel_ingot.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "icon": "minebound:illuminant_steel_ingot",
+    "category": "minebound:alloys",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "$(o)Steel is iron on coke, and thanks to the innately magical properties of the nether's very own light source, alloying coal and glowstone with energized iron results in an even stronger metal with better energy conduction!"
+        }, 
+        {
+            "type": "patchouli:text",
+            "text": "Craft in an $(l:minebound:blocks/alloy_furnace)alloy furnace$() by smelting an $(l:minebound:items/energized_iron) with coal and glowstone dust."
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/illuminant_steel_ingot.json:12]: The `minebound:items/energized_iron` link is bad.

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/myrial_steel_ingot.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "category": "minebound:items",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "$(o)Unsurprisingly, the my'ri found a metal as capable as conducting their powers as they are. The issue is that it's extremely durable, requiring something more powerful than diamond to extract. While netherite is a chore to find, at least myrial steel is plentiful.$()$(br2)Found as blocks in $(l:minebound:structures/crashed_ship)crashed ships$() and $(l:minebound:structures/ruined_base)ruined bases.$() Requires netherite to mine."
+        }, 
+        {
+            "type": "patchouli:crafting",
+            "recipe": "myrial_steel_block"
+        }
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/myrial_steel_ingot.json:13]: This recipe isn't showing up.

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/blocks/myrial_locker.json
Comment (MattKirchh) [inline src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/blocks/myrial_locker.json:1]: Copy/paste fail? This is identical to `myrial_console` when it should be about the locker.

**Review by MattKirchh — CHANGES_REQUESTED** [review]

**Comment by MattKirchh** [comment]
I'm guessing we'd need to do something custom to show the ingot recipes in the alloy furnace?

**Comment by MattKirchh** [comment]
> I'm guessing we'd need to do something custom to show the ingot recipes in the alloy furnace and recipes in the armor forge?

Based on some limited research, actually doesn't look too difficult. Just need to make a template for it.

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/myrial_diamond_ingot.json (hunk 1 above)
Comment (insertStringHere) [reply src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/myrial_diamond_ingot.json]: fixed

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/illuminant_steel_ingot.json (hunk 2 above)
Comment (insertStringHere) [reply src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/items/illuminant_steel_ingot.json]: fixed

File: src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/blocks/myrial_locker.json
Comment (insertStringHere) [reply src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/blocks/myrial_locker.json:1]: fixed

Comment (MattKirchh) [reply src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/blocks/myrial_locker.json:1]: The recipe is still for the console.

Comment (insertStringHere) [reply src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/blocks/myrial_locker.json:1]: fixed

**Review by MattKirchh — APPROVED** [review]

**Review by YonatanWN — APPROVED** [review]

**Pull request merged**